src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -108,6 +108,12 @@
   <module name="org.openhab.tools.analysis.checkstyle.PomXmlCheck">
     <property name="severity" value="error" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.PackageExportsNameCheck">
+    <property name="severity" value="error" />
+    <property name="sourceDirectories" value="src\main\java" />
+    <property name="excludedPackages" value=".*.internal.*, .*.test.*" />
```

Comment:
A good point. This was mainly targeted at test bundles, but when I think again, maybe it will be better to disable this check on test bundles at all and to remove this exclude.